runtime/src/iree/builtins/ukernel/arch/x86_64/BUILD

```diff
@@ -0,0 +1,41 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+load("//build_tools/bazel:build_defs.oss.bzl", "iree_runtime_cc_library")
+
+package(
+    default_visibility = ["//visibility:public"],
+    features = ["layering_check"],
+    licenses = ["notice"],  # Apache 2.0
+)
+
+iree_runtime_cc_library(
```

Comment:
since all these are header-only you can simplify this into a single target to reduce cmake overheads (and when we build this for the compiler we're going to want to glob anyway)

Reply:
This is only the BUILD file, and it the CMakeLists is hand-authored, not generated from that.

This BUILD file is just a stub at the moment. Whatever allows the build to continue with optimized code paths gracefully disabled. It would not be hard to support in Bazel, just some annoying `select` and `config_setting` business.